docs/copied-from-beats/security/basic-auth.asciidoc

```diff
@@ -0,0 +1,95 @@
+[role="xpack"]
+[[beats-basic-auth]]
+=== Configuring Authentication Credentials for {beatname_uc}
+
+When sending data to a secured cluster through the `elasticsearch`
+output, {beatname_uc} must either provide basic authentication credentials
+or present a client certificate.
+
+To configure authentication credentials for {beatname_uc}:
+
+. Create a role that has the `manage_index_templates` and
+`monitor` cluster privileges, and `read`, `write`, and `create_index`
+privileges for the indices that {beatname_uc} creates. You can create roles from the
+**Management / Roles** UI in {kib} or through the `role` API.
+For example, the following request creates a ++{beat_default_index_prefix}_writer++ role:
++
+["source","sh",subs="attributes,callouts"]
+---------------------------------------------------------------
+POST _xpack/security/role/{beat_default_index_prefix}_writer
```

Comment:
Since this is APM Server documentation, might worth be explicit about that this call is made to ES. There is nothing wrong neither with writing a full example that users can copy & paste like
`curl -u elastic:changeme -H "Content-Type: application/json" -X POST http://localhost:9200/_xpack/security/role/...`

Reply:
great point, I'll think about how to refactor the docs so we can put that in.  Maybe just a header before we include this page.